### DISCUSSION TITLE
RBAC not working with files service

### DISCUSSION THREAD

**Discussion by flower-of-the-bridges** [mia-platform/community#320] [Q&A] [answered]
### Which section?

Console

### Which is the version?

v11.4.2

### Description

Hi

I'm trying to release the `rbac-service` sidecar with the `files-service:2.9.1` and I obtain the following error:

```
panic: missing environment variables, one of API_PERMISSIONS_FILE_PATH or TARGET_SERVICE_OAS_PATH is required
```

Why this variable is not set?

**Comment by davidebianchi** [comment, marked as answer]
Hi @flower-of-the-bridges!

There is at least a manual routes configured? Or the documentation path of the service, but the files-service does not have the rond configuration set.

**Reply by flower-of-the-bridges** [reply]
Hi @davidebianchi , yes I didn't have both 😆 

After adding the documentation path and at least one manual route, the sidecar started

Thanks!!

**Reply by davidebianchi** [reply]
Perfect! One of the two info are needed.
